engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/meta/EqualsNode.java

```diff
@@ -44,11 +45,7 @@ public static EqualsNode build() {
 
   public abstract boolean execute(@AcceptsError Object self, @AcceptsError Object right);
 
-  /**
-   * Primitive values
-   **/
-
-
+  /** Primitive values */
```

Comment:
This used to be a 'section' comment, but now reformatting seems to have attached it to the `equalsBoolBool` method.

That doesn't seem right.

I think we should either keep these as 'sections' or remove them. Being attached to the first method is just awkward.

Reply:
You are right. I will remove it. Recently, I enabled a plugin in my IDE that does automatic reformatting on commits and this is the result.